cli/oyster-cvm/src/main.rs

```diff
@@ -36,12 +36,20 @@ enum Commands {
         docker_compose: String,
 
         /// List of Docker image .tar file paths
-        #[arg(short = 'i', long, required = true)]
+        #[arg(short = 'i', long, default_value = "")]
         docker_images: Vec<String>,
 
         /// Output folder name
         #[arg(short, long, default_value = "result")]
         output: String,
+
+        /// Git commit reference for oyster-monorepo
+        #[arg(
+            short = 'r',
+            long,
+            default_value = "3e6dbc844b42281462e65d7742d9436d4205fcfd"
```

Comment:
why is this exact commit used as default? need comment to instruct if this needs to be changed in future. When does a commit qualify for a default value here.

Reply:
any commit with the current nix configs will be relevant. Will have to be changed incase we make any changes to nix configs (add any param / change the project structure).
Will add a comment in the next pr.

Reply:
done